openpype/modules/deadline/plugins/publish/submit_max_deadline.py

```diff
@@ -110,9 +111,14 @@ def get_job_info(self):
             "AVALON_TASK",
             "AVALON_APP_NAME",
             "OPENPYPE_DEV",
-            "OPENPYPE_VERSION",
```

Comment:
@antirotor is this ok to be removed?  I have added the code which was used in other hosts.

Reply:
Yeah, looks good to remove.

Reply:
This should be removed as its a separate PR now.